src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBaseScrollContentPresenter.cs

```diff
@@ -60,6 +60,18 @@ public ScrollBarVisibility VerticalScrollBarVisibility
 			}
 		}
 
+		public bool CanHorizontallyScroll
+		{
+			get => HorizontalScrollBarVisibility != ScrollBarVisibility.Disabled;
+			set { }
```

Comment:
?

Reply:
@davidjohnoliver  This property is part of the public contract on the SCP (including the `set` :( ), while the `HorizontalScrollBarVisibility` is not. As I was needing this, I've update the ISCP interface to better reflect the SCP contract, but didn't changed the implementation and only re-route to the old property.